modules/ivc_champva/app/models/ivc_champva/vha_10_10d_2027.rb

```diff
@@ -24,9 +24,9 @@ def metadata # rubocop:disable Metrics/MethodLength
         'veteranMiddleName' => @data.dig('veteran', 'full_name', 'middle'),
         'veteranLastName' => @data.dig('veteran', 'full_name', 'last'),
         'veteranEmail' => @data.dig('veteran', 'email'),
-        'sponsorFirstName' => @data.fetch('applicants', [])&.first&.dig('full_name', 'first'),
-        'sponsorMiddleName' => @data.fetch('applicants', [])&.first&.dig('full_name', 'middle'),
-        'sponsorLastName' => @data.fetch('applicants', [])&.first&.dig('full_name', 'last'),
+        'sponsorFirstName' => @data.fetch('applicants', [])&.first&.dig('applicant_name', 'first'),
+        'sponsorMiddleName' => @data.fetch('applicants', [])&.first&.dig('applicant_name', 'middle'),
+        'sponsorLastName' => @data.fetch('applicants', [])&.first&.dig('applicant_name', 'last'),
```

Comment:
This may be a bug that I discovered. There isn't a full_name attribute in the fixture JSON; however, there is an applicant_name attribute that is being used instead.

Either the fixture is incorrect or the model is